angular-ngrx-scss/src/app/state/profile/profile.reducer.ts

```diff
@@ -14,10 +14,11 @@ const reducer = createReducer(
     ...state,
     ...data,
   })),
-  on(setSortAndFilterProperties, (state, { filters: { search } }) => ({
+  on(setSortAndFilterProperties, (state, { filters: { search, type } }) => ({
     ...state,
     sortAndFilter: {
       search,
+      type,
     },
```

Comment:
[curious] will this action always be called with search and type props? if yes, it's fine, if not, do we want to spread an existing state before updating?

Reply:
removed the spread, realize just passing the filters received is Ok to update the sort&Filters property.